chart/compass/charts/external-services-mock/templates/virtual-service.yaml

```diff
@@ -50,6 +50,31 @@ spec:
 ---
 apiVersion: networking.istio.io/v1alpha3
 kind: VirtualService
+metadata:
+  name: {{ template "fullname" . }}-external-services-mock-external-mtls-global-ord
```

Comment:
Can you add `-registry` here and everywhere on this PR to the `external-services-mock-external-mtls-global-ord` name?

So it would be `external-services-mock-external-mtls-global-ord-registry`

Reply:
It makes it more clear what the component is